tiledb/ml/readers/pytorch_sparse.py

```diff
@@ -98,6 +99,43 @@ def __check_row_dims(self, x_row_idx: np.ndarray, y_row_idx: np.ndarray) -> None
                 "of TileDB arrays X, Y should be of equal domain extent inside the batch."
             )
 
+    def __to_csr(
+        self, array_id: str, buffer: Dict[str, np.array], offset: int
+    ) -> csr_matrix:
+        """
+        :param array_id: The matrix on which the transformation will have effect 'X' for x_array and 'Y' for y_array
+        :param buffer: The buffered slice of the matrix to be batched
```

Comment:
Neither the name "buffer" nor the docstring seem accurate and descriptive to me for a parameter  mapping strings to arrays. Btw `Mapping` is a better annotation than `Dict` for read-only mappings.

Reply:
Semantically buffer is indeed buffer. The only reason why it is type annotated like that is that the returned slice from tiledb is and `OrderedDict` in `COO` format meaning records of 'features: [], dim0: ..[] , dim1: [] etc'